src/test/java/BaseTest.java

```diff
@@ -100,3 +162,10 @@ public static Object[][] getDataFromDataproviders() {
         };
     }
 }
+
+//Fluent wait example
+//FluentWait wait=new FluentWait(LoginTests.drive)
+//wait
+//.withTimeout(Duration.ofSeconds(10))
+//.pollingEvery(Duration.ofSeconds(1))
+//.ignoring(Exception.class)
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.